simuleval/evaluator/instance.py

```diff
@@ -201,6 +201,15 @@ def receive_prediction(self, prediction: TextSegment):
         else:
             raise NotImplementedError
 
+        if prediction.incomplete_word:
+            first_half = self.prediction_list[-1]
+            second_half = prediction_list[0]
+            complete_word = first_half + second_half
+            self.prediction_list.pop()
+            self.delays.pop()
```

Comment:
Could we also pop the last from `self.elapsed`? It's similar to `self.delays` except that it also includes the actual inference time